runtime/filetype.lua

```diff
@@ -3,7 +3,7 @@ if vim.g.did_load_filetypes then
 end
 vim.g.did_load_filetypes = 1
 
-vim.api.nvim_create_augroup('filetypedetect', { clear = false })
+vim.api.nvim_create_augroup('nvim.filetypedetect', { clear = false })
 
 vim.api.nvim_create_autocmd({ 'BufRead', 'BufNewFile', 'StdinReadPost' }, {
   group = 'filetypedetect',
```

Comment:
This should match the groupname above

Reply:
I decided to not modify this group after all. Docs and tests rely on this name and it seems to come from vim.